atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1363,12 +1363,32 @@ void WebContents::DidChangeVisibleSecurityState() {
   blink::WebSecurityStyle security_style =
     web_contents()->GetDelegate()->GetSecurityStyle(
       web_contents(), &explanations);
-    if (explanations.displayed_mixed_content &&
-        security_style == blink::kWebSecurityStyleNeutral) {
+
+  SecurityStateTabHelper* helper =
+    SecurityStateTabHelper::FromWebContents(this);
+  DCHECK(helper);
+  security_state::SecurityInfo security_info;
+  helper->GetSecurityInfo(&security_info);
+
+  if (explanations.displayed_mixed_content &&
+      security_style == blink::kWebSecurityStyleNeutral) {
       Emit("security-style-changed", "passive-mixed-content");
```

Comment:
related to this PR `blink::WebSecurityStyle` can be passed directly to the Emit because it has a converter (`insecure` needs to be changed to `passive-mixed-content` to match browser-laptop). Rather than adding the ev_string below, we should create a similar converter for the SecurityInfo to pass all info in a JS object. The ev_string can be created in browser-laptop from the raw info